components/landing/TheQiskitCapabilitiesSection/QiskitCapabilityCard.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <article class="qiskit-capability-card">
+    <div class="qiskit-capability-card__container">
+      <div class="qiskit-capability-card__thumbnail">
+        <img class="qiskit-capability-card__thumbnail__media" :src="thumbnailResource">
+      </div>
+      <div class="qiskit-capability-card__copy">
+        <h3 class="qiskit-capability-card__title">
+          {{ title }}
+        </h3>
+        <img class="qiskit-capability-card__thumbnail__mobile" :src="thumbnailResource">
+        <div class="qiskit-capability-card__description">
+          <p v-text="description" />
+          <AppCta
+            v-if="link"
+            v-bind="link"
+            kind="ghost"
+          />
+        </div>
+      </div>
+    </div>
+  </article>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
+
+@Component
+export default class QiskitCapabilityCard extends Vue {
+  @Prop({ type: String, required: true }) thumbnailResource!: string
+  @Prop({ type: String, required: true }) title!: string
+  @Prop({ type: String, required: true }) description!: string
+  @Prop({ type: Object, required: false }) link!: any
+}
+</script>
+
+<style lang="scss" scoped>
+.qiskit-capability-card {
+  display: flex;
+
+  @include mq($until: medium) {
+    flex-direction: column;
+  }
+
+  &__container {
+    display: flex;
+
+    @include mq($until: large) {
+      flex-direction: column;
+    }
+  }
+
+  &__copy {
+    padding-right: $spacing-07;
+  }
+
+  &__thumbnail {
+    width: 100%;
+    max-width: 10rem;
```

Comment:
I think we can fix the sizes of the images themselves to `10rem` instead of having multiple instances of `min-width`, `max-width` and `width: 100%;`, because I believe that these images are always `10rem` anyway, as long as the user doesn't use a 160px wide screen device (or smaller), which sounds quite unlikely in this day and age.